### DISCUSSION TITLE
Homebox deployment Feb 2024 (AKA: "the great home organization project") - pics only

### DISCUSSION THREAD

**Discussion by danielrosehill** [hay-kot/homebox#841] [Show and tell]
![20240214_145413](https://github.com/hay-kot/homebox/assets/40233203/a5013fe5-3aea-4f1b-80a6-aa5e820fb2be)
![20240214_145618](https://github.com/hay-kot/homebox/assets/40233203/534216f8-70a1-401e-98d6-5d6f5ad4c79e)
![20240214_150047](https://github.com/hay-kot/homebox/assets/40233203/e5b4b3cf-cfe8-4dc0-862c-de11635b725f)
![20240218_234304](https://github.com/hay-kot/homebox/assets/40233203/5549b1c4-25f6-4b51-91d0-fd433495662f)
![20240221_014637](https://github.com/hay-kot/homebox/assets/40233203/5865cb77-4fe4-46f2-98e1-be457d995a19)
![20240223_154255](https://github.com/hay-kot/homebox/assets/40233203/9c0d8614-8887-4b40-8d76-0c1c6971fbca)
![20240223_154450](https://github.com/hay-kot/homebox/assets/40233203/b0251cbf-3e13-4f8d-bf86-68232211261a)
![20240223_154455](https://github.com/hay-kot/homebox/assets/40233203/a3ad14d6-ef8b-480c-947d-9fed9f6bbcde)
![20240307_170926](https://github.com/hay-kot/homebox/assets/40233203/6436c6c1-beca-493b-ae3f-8dc9f0b2a713)
![IMG_20240216_030021](https://github.com/hay-kot/homebox/assets/40233203/c0ab65ea-01ba-4d9e-be56-515c14558485)
![IMG_20240216_030044](https://github.com/hay-kot/homebox/assets/40233203/d0516e77-e191-460d-b42f-2fab58e1824f)
![IMG_20240224_164619](https://github.com/hay-kot/homebox/assets/40233203/b62c3b58-fb33-4fa4-81cd-01372e42c9e1)
![IMG_20240227_145423](https://github.com/hay-kot/homebox/assets/40233203/c4d6280f-3ec9-4118-9930-2b51443e4c6b)
![IMG_20240301_154339](https://github.com/hay-kot/homebox/assets/40233203/be063645-c79d-4f12-bbae-595ba1606924)
![IMG_20240301_154349](https://github.com/hay-kot/homebox/assets/40233203/97446afd-21ea-4b05-aaf4-13888f2956b4)
![IMG_20240301_175703](https://github.com/hay-kot/homebox/assets/40233203/ef763a22-d744-4618-8d92-643ce523842d)
![IMG_20240302_191507](https://github.com/hay-kot/homebox/assets/40233203/1b7c6fea-7001-4e08-a9d1-91590a649f6a)
![IMG_20240302_233639](https://github.com/hay-kot/homebox/assets/40233203/6c3e8cbb-c01c-4576-bada-6a0b0350a3ac)
![IMG_20240303_151055](https://github.com/hay-kot/homebox/assets/40233203/90073ad6-e8c3-4f3d-9a29-8a6142c7a689)
![IMG_20240304_230912](https://github.com/hay-kot/homebox/assets/40233203/f076cabe-d214-4491-8430-326dc6f7e759)
![IMG_20240307_030001](https://github.com/hay-kot/homebox/assets/40233203/4de8ca3b-cd8b-48f3-8d4f-38080595864c)
![IMG_20240307_050907](https://github.com/hay-kot/homebox/assets/40233203/39774cbf-93c0-49ca-aac2-21809dcc8d04)
![IMG_20240307_060544](https://github.com/hay-kot/homebox/assets/40233203/36c4437b-f73d-4a6f-9a18-6b45a1455869)
![IMG_20240307_073439](https://github.com/hay-kot/homebox/assets/40233203/4150d7a3-9ead-418a-879f-625357df63d4)
![IMG_20240307_170536](https://github.com/hay-kot/homebox/assets/40233203/99ffb784-39be-4bf9-8dd1-f35fe7c2b8fd)
![IMG_20240307_170637](https://github.com/hay-kot/homebox/assets/40233203/fb2988b5-8e90-444b-b1eb-634dde190654)
![IMG_20240307_170805](https://github.com/hay-kot/homebox/assets/40233203/7a7a3288-0119-453d-b158-eed4f21398f8)
![IMG_20240307_171202](https://github.com/hay-kot/homebox/assets/40233203/4a43c463-c5c7-4096-8c04-d9377958e133)
![IMG_20240307_171653](https://github.com/hay-kot/homebox/assets/40233203/7f0c367c-5b63-4085-9e16-5201b29d93c8)
![IMG_20240307_172314](https://github.com/hay-kot/homebox/assets/40233203/92ab5012-fc01-4806-a1e0-422ec246d755)
![IMG_20240307_201755](https://github.com/hay-kot/homebox/assets/40233203/3757e743-f513-4cb4-9859-5c04e25530fa)

**Comment by legrego** [comment]
Wow, great job! What worked well for you with respect to homebox? Did you face any challenges?

**Comment by danielrosehill** [comment]
Hi Larry!

Wonderful system overall.

I would say that the biggest impediment to use currently is an upstream bug
that occasionally leaves old data on the screen unless you manually refresh
the tab.

It makes it VERY easy to accidentally overwrite data and having to refresh
constantly is annoying.

I'm confident they'll get it fixed. But that's the only caveat I would
attach to my current recommendation.

I'm more than 3,000 items in and it's performing very robustly (and if I
can say so myself, my backup scripts are working nicely too to get
everything backed up).

Regards,
---------
*Daniel Rosehill*
Homepage <http://www.danielrosehill.com>

**Comment by justin-pierce** [comment]
@danielrosehill how does the barcode scanner work? saw mention of 'inventory mode' -- what happens when you scan something and how does it fit in your workflows?
